src/biotite/structure/transform.py

```diff
@@ -234,6 +234,78 @@ def rotate_about_axis(atoms, axis, angle, support=None):
     return _put_back(atoms, positions)
 
 
+def orient_principal_components(atoms):
+    """
+    Translate and rotate the atoms to be centered at the origin with the
+    principal axes aligned to x, y, and z.
+
+    Parameters
+    ----------
+    atoms : AtomArray or ndarray, shape=(n,3)
+        The atoms of which the coordinates are transformed.
+        The coordinates can be directly provided as :class:`ndarray`.
+
+    Returns
+    -------
+    AtomArray or ndarray, shape=(n,3)
+        The atoms with coordinates centered at the orgin and aligned with
+        xyz axes.
+
+    See Also
+    --------
+    rotate_centered
+    rotate_about_axis
+
+    Examples
+    --------
+    Align principal components to xyz axes
+
+    >>> print("original variance =", molecule.coord.var(axis=0))
+    original variance = [ 4.8955374  2.3114026 16.263195 ]
+    >>> moved = orient_principal_components(molecule)
+    >>> print("moved variance =", moved.coord.var(axis=0))
+    moved variance = [21.425678   1.1962254  0.8482297]
```

Comment:
This doctest fails in the CI, since `molecule` is not known. However, `atom_array` is, based on PDB 1L2Y. I am not sure though, whether the resulting numbers are correct in this case.

```suggestion
    >>> print("original variance =", atom_array.coord.var(axis=0))
    original variance = [ 4.8955374  2.3114026 16.263195 ]
    >>> moved = orient_principal_components(atom_array)
    >>> print("moved variance =", moved.coord.var(axis=0))
    moved variance = [21.425678   1.1962254  0.8482297]
```

The other failing test is not linked to your PR, but is caused by a change in the PDB.